content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,63 +2,128 @@
 Getting Started
 ===============
 
-Helpdesk teams provide your customers with support to queries or errors they might encounter while
-using your product/service. Therefore, a successful scheme where you can organize multiple teams
-with their customized pipeline, visibilities settings, and ticket traceability is essential.
+Helpdesk teams provide customers with support to queries or issues they might encounter while using
+a product/service. Therefore, having a system in place where multiple teams can co-exist with their
+own customized pipeline, visibility settings, and ticket traceability is essential.
 
 Set up teams
 ============
 
-| To modify or create teams, go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`.
-| Setting up multiple teams allows you to group tickets by your channels (example: BE/US), or by
-  your support services' types (example: IT, accounting, admin, etc.).
+Setting up multiple teams allows for tickets to be grouped by location
+(example: :abbr:`BE/US(Belgium/United States)` ), or by support type (example: IT, accounting, etc.).
 
-.. image:: getting_started/helpdesk_teams_view.png
+To modify or create teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`.
+
+.. image:: getting_started/helpdesk_teams_list.png
    :align: center
    :alt: View of the helpdesk teams page in Odoo Helpdesk
 
-Team’s productivity and visibility
-----------------------------------
+Assignments and visibility
+--------------------------
+
+Visibility
+~~~~~~~~~~
+The *Visibility* feature specifies who can view this team:
```

Comment:
Heading can be more descriptive.
Also need a line break after the heading in the RST.

For the intro paragraph, since we're directly referencing the UI here then it gets a `:guilabel:`. As well, it's not really a feature as much as it's a _setting_. Lastly, no colon before list items that also have colons; just end the sentence on a period. 

Here's how I would rewrite this whole section.

```suggestion
Visibility
~~~~~~~~~~

Under the :guilabel:`Visibility` subheading, decide who can view this team and their activities.
```